changelog.md

```diff
@@ -14,6 +14,14 @@
   ```
   will no longer compile.
 
+- In template and generic bodies, for symbols with only 1 overload in scope,
+  previously the compiler would not allow other overloads in the instantiation
+  context to be used, which was not the case for 0 or more than 1 overloads.
+  Now other overloads are allowed with a preference for the original overload
+  in cases of ambiguity. Some code might need to be changed to use `bind` to
```

Comment:
Since this can be breaking (required package PRs https://github.com/Nycto/DelaunayNim/pull/5 and https://github.com/PMunch/nimlsp/pull/168), we could make it `experimental`.